src/Silverback.Integration/Messaging/Configuration/ProducerEndpointBuilderEncryptUsingExtensions.cs

```diff
@@ -28,13 +28,17 @@ public static class ProducerEndpointBuilderEncryptUsingExtensions
         ///     The optional initialization vector (IV) for the symmetric algorithm. If <c>null</c> a different IV
         ///     will be generated for each message and prepended to the actual message payload.
         /// </param>
+        /// <param name="keyIdentifier">
+        ///     The key identifier.
```

Comment:
_An optional key identifier to be sent in the header. When rotating keys, it will be used on the consumer side to determine the correct key to be used to decrypt the message_
...or something similar.